src/aics-image-viewer/components/CellViewerCanvasWrapper/index.tsx

```diff
@@ -68,23 +89,23 @@ export default class ViewerWrapper extends React.Component {
 
 const STYLES = {
   viewer: {
-    display: "flex",
-    position: "relative",
+    display: "flex" as "flex",
+    position: "relative" as "relative",
```

Comment:
thank you typescript.

Reply:
I looked around for a little while for some type system magic that would convince TS to interpret all the string fields of an object as literal types. Maybe someday I'll learn that it's possible.

Reply:
this might be the best choice: https://stackoverflow.com/questions/46710747/type-string-is-not-assignable-to-type-inherit-initial-unset-fixe (use CSSProperties type)

Reply:
This makes sense to me. Though this would make the type of all the `STYLES` objects look like `{ [class: string]: React.CSSProperties }`. Is it reasonable to inline that everywhere it might appear, or is it worth defining that type in a single place as well?

Reply:
I think you would just do 
```
const myStyles: React.CSSProperties = {
  position: 'absolute',
}
```
and done.

Reply:
Sorry, I see that it's nested.  Yeah I believe you'd have to type the nested objects.

Reply:
you could use the postfix `as React.CSSProperties;` as well.  I think they are equally readable in this scenario.